experimental/addons/mongodb/template.cue

```diff
@@ -0,0 +1,65 @@
+package main
+
+
+output: {
+        apiVersion: "core.oam.dev/v1beta1"
+        kind:       "Application"
+        spec: {
+                components: [
+                        {
+                                type: "k8s-objects"
+                                name: "mongodb-ns"
+                                properties: objects: [{
+                                        apiVersion: "v1"
+                                        kind:       "Namespace"
+                                        metadata: name: parameter.namespace
+                                }]
+                        },
+                        {
+                            name: "mongodb-helm"
+                            type: "helm"
+                            dependsOn: ["mongodb-ns"]
+                            type: "helm"
+                            properties: {
+                                    repoType: "helm"
+                                    url:      "https://charts.bitnami.com/bitnami"
+                                    chart:    "mongodb"
+                                    targetNamespace: parameter["namespace"]
+                                    version:  "13.1.2"
+                                    values: {
+                                            persistence: {
+                                                enabled: parameter.persistenceEnabled
+                                            }
+                                            if parameter.persistenceEnabled {
+                                                persistence: {
+                                                    storageClass: parameter.persistenceStorageClass
+                                                }
+                                            }
+                                    }
```

Comment:
In my view, you should make it as a ComponentDefinition.

Install this addon just install this definition without installing the helm chart of mongodb.

When users want to create a mongodb, they just choose this component type and create an application.

If you're installing a mongodb-operator, then this is a right choice to install it here.